recipeapp/src/Recipes/index.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import SearchBar from './SearchBar/SearchBar';
+import '../Recipes/recipes.css'
```

Comment:
This can be shorter. Also, you're missing a semicolon

```JavaScript
import './recipes.css';
```

Reply:
I have made the changes.